app-admin/kubelet-wrapper/files/kubelet-wrapper

```diff
@@ -31,59 +31,48 @@ if [[ -n "${KUBELET_VERSION}" ]]; then
 fi
 
 if [[ -n "${KUBELET_ACI}" ]]; then
-	echo KUBELET_ACI environment variable is deprecated, please use the KUBELET_IMAGE_URL instead
+	echo KUBELET_ACI environment variable is deprecated. This script uses Docker now, please use KUBELET_IMAGE_URL instead and pass a Docker image name
 fi
 
 if [[ -n "${RKT_OPTS}" ]]; then
-	echo RKT_OPTS environment variable is deprecated, please use the RKT_RUN_ARGS instead
+	echo RKT_OPTS environment variable is deprecated, this script uses Docker now, please use DOCKER_RUN_ARGS instead and pass Docker options
+fi
+
+if [[ -n "${RKT_RUN_ARGS}" ]]; then
+	echo RKT_RUN_ARGS environment variable is deprecated, this script uses Docker now, please use DOCKER_RUN_ARGS instead and pass Docker options
```

Comment:
Lokomotive also fails with this change because $RKT_RUN_ARGS is ignored. Symptoms:
```
Nov 24 16:55:30 ip-10-0-45-152 kubelet-wrapper[21293]: RKT_RUN_ARGS environment variable is deprecated, this script uses Docker now, please use DOCKER_RUN_ARGS instead and pass Docker options
...
Nov 24 16:55:31 ip-10-0-45-152 kubelet-wrapper[21293]: F1124 16:55:31.580198       1 kubelet.go:1413] failed to start OOM watcher open /dev/kmsg: no such file or directory
Nov 24 16:55:31 ip-10-0-45-152 systemd[1]: kubelet.service: Main process exited, code=exited, status=255/EXCEPTION
Nov 24 16:55:31 ip-10-0-45-152 systemd[1]: kubelet.service: Failed with result 'exit-code'.
```

I am wondering if Flatcar needs to support both kubelet-wrapper with rkt and Docker for a little while, so that Typhoon, Lokomotive and other Kubernetes that depend on that script have some time to adapt, since it's a breaking change.

Otherwise, Kubernetes distros would need to somehow detect whether to use RKT_RUN_ARGS or DOCKER_RUN_ARGS.